### PR TITLE
[MRI_violation] clarification to instruction and few grammatical corrections

### PR DESCRIPTION
**Pull request by PapillonMcGill**
## Brief summary of changes
added some clarifications to the test plan

### CODE REVIEW

**Comment by christinerogers** [comment]
a side thought - Would a mention of checking for Help text be too much here?  
We just realized (#5714) these 2 module sub-pages have been missing Help content for a very long time.
This would've been caught long ago any Test plan had included (e.g.) "Is the Help text working and up-to-date?")
As well, #5709 (subpage help not showing) would have been caught weeks if not months ago.   
I'll raise for discussion next week.

**Comment by PapillonMcGill** [comment]
@christinerogers I added check for help content. Is the text as currently worded fine with you?
I could change for `... content specific about ...` if you wish for sub-pages.

**Comment by christinerogers** [comment]
> @christinerogers I added check for help content. Is the text as currently worded fine with you?
> I could change for `... content specific about ...` if you wish for sub-pages.

@PapillonMcGill It looks good to me like that, but yes, a really brief content review would be nice to suggest too.  
I spot a typo in `rigth` on all 3 new lines - maybe at the same time add `and is up-to-date` ?

File: modules/mri_violations/test/TestPlan.md [hunk 1]
```diff
@@ -4,43 +4,54 @@
...
-4. Resolve issues with each of the different types of resolutions, and ensure 
-    that they show up in the `Resolved` tab, and are no longer in the 
-    `Not resolved` tab.
-5. Ensure that the link to the protocol violation works within the `Resolved` 
-    tab.
+4. Go back to the MRI_violations page and resolve issues with each of the
+    different types of resolutions, and ensure that they show up in the 
+    `Resolved` tab, and are no longer in the `Not resolved` tab.
+5. Ensure that the link to the `protocol violation` and `Could not identify scan
+   type' works within the `Resolved` tab (`Problem` column).
```
Comment (johnsaigle) [inline modules/mri_violations/test/TestPlan.md]:
```suggestion
   type` works within the `Resolved` tab (`Problem` column).
```

The file isn't rendering properly but this should fix it.

**Pull request merged**